base/libgit2/diff.jl

```diff
@@ -34,6 +34,26 @@ function diff_tree(repo::GitRepo, oldtree::GitTree, newtree::GitTree)
     return GitDiff(repo, diff_ptr_ptr[])
 end
 
+function GitDiffStats(diff::GitDiff)
```

Comment:
I don't think so. I grepped for it. This is the way you're supposed to extract stats from the diff, I think, according to the API docs.